iosApp/PhongKMMIC/Sources/Presentation/Modules/SurveyDetail/SurveyDetailCombineViewModel.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  SurveyDetailCombineViewModel.swift
+//  PhongKMMIC
+//
+//  Created by Phong Vo on 17/05/2023.
+//  Copyright © 2023 Nimble. All rights reserved.
+//
+
+import Combine
+import Factory
+import KMPNativeCoroutinesCombine
+import shared
+
+final class SurveyDetailCombineViewModel: ObservableObject {
+
+    @Published var isLoading = false
+    @Published var errorMessage: String?
+    @Published var backgroundImageUrl: String = ""
```

Comment:
When we use directly value from view model and when we use UiModel on view?

Reply:
Solved in ecb0b79